main/REPL/scratch.md

```diff
@@ -0,0 +1,36 @@
+You use `scratch` to get, set, and list to save key-value pairs for later. It is only on the ag-solo and is not accessible from the chain, making it private to the ag-solo user. Since deploy scripts are ephemeral, use scratch to save objects in a deploy script for later scripts to use.
```

Comment:
`get, set, and list to save` seems like a weird phrasing. What about:

```suggestion
You use `scratch` to save key-value pairs for later. It is only on the ag-solo and is not accessible from the chain, making it private to the ag-solo user. Since deploy scripts are ephemeral, use scratch to save objects in a deploy script for later scripts to use.
```